src/Features/OnboardingTasks.php

```diff
@@ -735,4 +787,26 @@ public function add_task_dismissal( $task_lists ) {
 
 		return $task_lists;
 	}
+
+	/**
+	 * Add the snoozed status to each task.
+	 *
+	 * @param array $task_lists Task lists.
+	 * @return array
+	 */
+	public function add_task_snoozed( $task_lists ) {
+		$snoozed_tasks = get_option( 'woocommerce_task_list_remind_me_later_tasks' );
+
+		foreach ( $task_lists as $task_list_key => $task_list ) {
+			foreach ( $task_list['tasks'] as $task_key => $task ) {
+				if ( isset( $task['isSnoozeable'] ) && in_array( $task['id'], array_keys( $snoozed_tasks ), true ) ) {
+					$task_lists[ $task_list_key ]['tasks'][ $task_key ]['isSnoozed']    = true;
```

Comment:
This bool can be dynamically created.   I haven't tested this, but something along the lines of:

```suggestion
					$task_lists[ $task_list_key ]['tasks'][ $task_key ]['isSnoozed']    = $snoozed_tasks[ $task['id'] ] > time();
```

Reply:
Ah, understand what you mean here now. Done in https://github.com/woocommerce/woocommerce-admin/pull/7539/commits/1979e575c0431127490c2ded6dffc98872bd22dc.